.github/stale.yml

```diff
@@ -0,0 +1,63 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 60
+
+# Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 10
+
+# Only issues or pull requests with all of these labels are check if stale. Defaults to `[]` (disabled)
+onlyLabels: []
+
+# Issues or Pull Requests with these labels will never be considered stale. Set to `[]` to disable
+exemptLabels:
+  - question/research
+  - feature/enhancement
+  - "waiting for dependency"
+  - "good first issue"
+  - priority
+
+# Set to true to ignore issues in a project (defaults to false)
+exemptProjects: false
+
+# Set to true to ignore issues in a milestone (defaults to false)
+exemptMilestones: false
+
+# Set to true to ignore issues with an assignee (defaults to false)
+exemptAssignees: true
```

Comment:
Maybe this also should be removed, imagine an issue assigned to someone and it is inactive for 2 months, I think staling it is like a reminder.

Reply:
I am not sure stale is a good reminder. If an issue is assigned by the maintainers, CTO should be the reminder. Don't start using stalebot as a reminder, it will mix the concepts in the long term. BTW it is a good idea to have a dashboard with the assigned issues I think there is a fancier tool for that then GitHub issue filtering.